packages/avro-ts/src/types/wrapped-union.ts

```diff
@@ -9,28 +9,32 @@ import { fullName } from '../helpers';
 const resolveItem = (context: Context, item: Schema): Schema =>
   typeof item === 'string' && context.refs?.[item] ? context.refs?.[item] : item;
 
-export const isWrappedUnion = (type: Schema, context: Context): type is schema.RecordType[] =>
+type WrappedUnionItem = schema.RecordType | 'null';
```

Comment:
could this also be extended to include other primitives?

Reply:
Maybe but is it a pattern that avsc actually uses I wonder?